### DISCUSSION TITLE
Midori browser anyone?

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#351] [General]
Have been dabbing around with the recently updated Midori browser on Windows and Linux, and i am impressed! 
Fast, syncs with Mozilla and find it to be lighter on resources than Firefox (even here on SpiralLinux). 
https://astian.org/midori-browser/

Btw, is there a good webpage for tips and tweaking SpiralLinux Cinnamon?
Like getting best sound and file content options etc.

SpiralLinux Cinnamon ed. is impressive! :)
That is a huge compliment for a basically KDE fan.

**Comment by dacomboman** [comment]
One "small" issue...
Didn't know it was in software repo. (flatpak only), had installed .deb but it cannot set it as "preferred application".
It fails to show up anywhere.
Maybe too soon to tell, but Midori does offer option to be set as default.
So, will see how that pans out when i get Thunderbird and accoutns set up.

**Reply by dacomboman** [reply]
Update: tested Midori as default with LibreOffice request for extensions, and it opened up as expected. Yay!

**Comment by AlphaElwedritsch** [comment]
I became aware of you here and started looking into it

See discussion here:
https://discuss.privacyguides.net/t/midori-browser-gecko-browser-for-desktop-android/15027/1

I would not recommend it
Not for daily use as main browser
possibly to observe further development. Maybe something useful and recommendable will come out of it...

**Comment by dacomboman** [comment]
You may be onto something there!
John Smith CEO & Founder - Okler
John Doe CTO & Founder - Okler
as per their website:
https://astian.org/about-us/

Hum...!

**Comment by dacomboman** [comment]
Can't be worse than Opera!
Besides, i really find it much faster than Firefox.
Have no plans of subscribing to any of Astian's services (at least for now).
